LSP.sublime-settings

```diff
@@ -570,7 +570,12 @@
         "Packages/R/R.sublime-syntax"
       ],
       "languageId": "r"
-    }
-
+    },
+    "sourcekit-lsp": {
+      "command": ["xcrun", "sourcekit-lsp"],
+      "scopes": ["source.swift"],
+      "syntaxes": ["Packages/Swift/Syntaxes/Swift.tmLanguage"],
+      "languageId": "swift"
+    },
```

Comment:
Looking at sourcekit-lsp's github page, it seems that this server can also handle the C-family of languages (C, C++, ObjC, ObjC++). This makes sense as it's based on clangd. Please add those languages. (See the clangd entry for guidance on how to add more than one language)

https://github.com/apple/sourcekit-lsp/tree/master/Editors

Reply:
sourcekit-lsp seems to directly call clangd, which also now ships with Xcode. Do you think I should add it anyways (knowing that if you want clangd, it's already available under clangd)? For example, I personally do not have clangd enabled because I cannot figure out a way to pass it compiler flags other than compile_flags.txt, which does not care for the project's language. So I would need to add this to my settings to disable all the "C" features because I can only enable all of the languages together:

```json
"sourcekit-lsp": {
	"enabled": true,
	"languages": [
		{
			"scopes": ["source.swift"],
			"syntaxes": ["Packages/Swift/Syntaxes/Swift.tmLanguage"],
			"languageId": "swift"
		}
	]
}
```

Reply:
OK sounds good to me. There are already at least two langservers handling the C-family anyway.